langgraph/channels/base.py

```diff
@@ -74,6 +75,13 @@ def get(self) -> Value:
 
         Raises EmptyChannelError if the channel is empty (never updated yet)."""
 
+    def consume(self) -> None:
```

Comment:
If we do that it means eg in the simple react graph with agent and tools, if you interrupt before tools and replace a message with tool calls with one without, you still go to tools channel afterwards because we didn't reset the branches

Reply:
ah i see ya